app/code/Magento/Webapi/Controller/Rest/SynchronousRequestProcessor.php

```diff
@@ -89,10 +91,17 @@ public function process(\Magento\Framework\Webapi\Rest\Request $request)
         $serviceClassName = $route->getServiceClass();
         $service = $this->objectManager->get($serviceClassName);
 
-        /**
-         * @var \Magento\Framework\Api\AbstractExtensibleObject $outputData
-         */
-        $outputData = call_user_func_array([$service, $serviceMethodName], $inputParams);
+        try {
+            /**
+             * @var \Magento\Framework\Api\AbstractExtensibleObject $outputData
+             */
+            $outputData = call_user_func_array([$service, $serviceMethodName], $inputParams);
+        } catch (\Exception $error) {
+            throw new Exception(new Phrase(
+                $error->getMessage()
+            ));
```

Comment:
Shouldn't we pass the original exception as previous?
```suggestion
            throw new Exception(
                new Phrase($error->getMessage()),
                previous: $e
            );
```

Example: https://3v4l.org/Smcri

Reply:
Yes, it's good point.